test/core/templaters/python_test.py

```diff
@@ -0,0 +1,25 @@
+"""Tests for templaters."""
+
+import pytest
+
+from sqlfluff.core.templaters import PythonTemplateInterface
+from sqlfluff.core import SQLTemplaterError
+
+
+PYTHON_STRING = "SELECT * FROM {blah}"
```

Comment:
Is there a reason you're using global variables rather than fixtures for the test strings?

Reply:
I'm using them because I'm using them in parameterized tests. Do you know if you can use fixtures in parameterised tests? If you do, then I'm totally up for it.

The idea would be to use one fixture for some tests and another fixture for others (or maybe some tests don't need the fixture).

Getting a good approach on this would be excellent to reduce bloat on some of the tests in `templaters.base_test,py` which is now _pretty ugly_.

Reply:
Are we talking about PyTest? If so, I'm pretty sure you can use fixtures in parameterized tests.

Reply:
Hmm, I just spend some time researching, and nothing turned up. Given that, I'm onboard with this approach